stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/LoadToTable.java

```diff
@@ -77,26 +68,12 @@
     @Override
     public void execute(Context context, CallableUnitCallback callback) {
         final String filePath = context.getStringArgument(0);
-        Path path;
```

Comment:
Don't we need the validation in the new implementation?

Reply:
@wggihan moved the functionality to IOUtils. Since this will be a duplicated method which will be reused by csv channel etc. In IOUtils we throw an exception if the validation fails. Hence we do the validation in that function.